### PR TITLE
[ICDS Dashboard] update pregnant report to use same month logic as lactating

### PR DESCRIPTION
**Pull request by calellowitz**
@Rohit25negi 
This applies the change you made to the lactating report, to also work for the pregnant one. Still determining if this addresses the underlying problem in https://dimagi-dev.atlassian.net/browse/ICDS-759 but i think it should.

### CODE REVIEW

**Comment by czue** [comment]
Guessing the test fail is real.
```
======================================================================
ERROR: custom.icds_reports.tests.reports.test_awc_reports:TestAWCReport.test_awc_report_lactating_fifth_record
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/icds_reports/tests/reports/test_awc_reports.py", line 2686, in test_awc_report_lactating_fifth_record
    data['data'][4],
IndexError: list index out of range
----------------------------------------------------------------------
```

File: custom/icds_reports/reports/awc_reports.py [hunk 1]
```diff
@@ -1236,7 +1237,7 @@ def get_pregnant_details(case_id, awc_id):
     'start', 'length', 'order', 'reversed_order', 'awc_id'
 ], timeout=30 * 60)
 def get_awc_report_lactating(start, length, order, reversed_order, awc_id):
-    latest_available_month = datetime.utcnow() - timedelta(days=1)
+    latest_available_month = date.today() - timedelta(days=1)
```
Comment (czue) [inline custom/icds_reports/reports/awc_reports.py]: for my own education, is this fixing anything or just being more consistent with other reports?

**Review by czue — APPROVED** [review]
don't have enough context on the logic change to have an opinion about the fix, but :white_check_mark: on code.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Looks good to me. @calellowitz  could you fix the test case?

File: custom/icds_reports/reports/awc_reports.py (hunk 1 above)
Comment (calellowitz) [reply custom/icds_reports/reports/awc_reports.py]: Just consistency, and because we dont actually need the time

**Pull request merged**